src/entrypoints/my-redirect.ts

```diff
@@ -65,6 +65,8 @@ const render = (showTroubleshooting: boolean) => {
   const redirectUrl =
     window.redirect.redirect === "oauth"
       ? `${instanceUrl}/auth/external/callback${params}`
+      : window.redirect.custom
+      ? `${instanceUrl}/${window.redirect.component}/_my_redirect/${window.redirect.redirect}${params}`
```

Comment:
Shouldn't we link to the my integration so we can also catch none installed components? And have more control?

Reply:
Actually, no I'm not fine with that.
a) We would not get more control (we would still require redirects to register in this repo).
b) The codebase of custom integrations does not have to follow the pase of core and vise-versa, which will make rolling out new paths much slower, and would require a lot of effort on all parts to make it compatible with as wide of a version span as possible.
c) Sure we get a component not loaded check, but nothing else, and with a and b, I don't see that single item way up against it.

Reply:
Extra control in that we could add an extra warning, check if the link works, check for versions of integrations, etc.

That rolling out new paths will be slower is not that big of an issue is it? Do we plan to add a lot of these?

Reply:
People are going to end up clicking on a badge, seeing official My Home Assistant page and end up in their instance with a 404 error page without any UI:

![image](https://user-images.githubusercontent.com/1444314/177859695-3d316a80-363c-451d-83dd-8d60f99347d1.png)

The My panel can handle this:

![image](https://user-images.githubusercontent.com/1444314/177859788-78398f4d-694e-4fc5-a61e-b50baf2b7dc0.png)

Reply:
https://github.com/home-assistant/frontend/pull/13153